README.md

```diff
@@ -2,6 +2,9 @@
 # Book Store
 
 > A single-page app that allows users store some of their favorite books. It lets users add their favorite books to the list and post them to an API. This project was built using Javascript, React-redux
+> Design followed [Zeplin](https://app.zeplin.io/projects) guidelines.
+
+![Bookstore](https://github.com/Meri-MG/bookstore---react/style/Bookstore.png)
 
```

Comment:
- [ ] What a wonderful design 🔥. It looks great 💯 . Just one requested change 🤔. Let's fix the path to this image as it's not displaying properly. This is very important as the README is the first thing our users see and we have to make a good impression 🥇. See it below:

<img width="525" alt="Loom_sd2g0p0zGQ" src="https://user-images.githubusercontent.com/81468675/150412008-b5c3c8f5-902f-4571-92a8-d2288a2a4d36.png">